index.js

```diff
@@ -0,0 +1,135 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+const URL_TRIP_BY_ID = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+
+//
+// Status Management
+//
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const loadTrips = () => {
+  reportStatus('Loading trips...');
+
+  const tripList = $('#trip-list');
+  tripList.empty();
+
+  axios.get(URL)
+  .then((response) => {
+    reportStatus(`Successfully loaded ${response.data.length} trips`);
+    tripList.append(`<tr><th>All Trips</th></tr>`);
+    response.data.forEach((trip) => {
+      tripList.append(`<tr><td id="${trip.id}" class="trip-item">${trip.name}</td></tr>`);
+    });
+    $('.trip-item').click( function(event) {
+      loadTripDetails(`${event.target.id}`);
+    });
```

Comment:
This ends up working, but we would prefer if you added your click event listener to each `.trip-item` using event delegation